packages/blade/src/components/Card/CardSurface.native.tsx

```diff
@@ -28,7 +28,7 @@ type CardSurfaceProps = {
   children: React.ReactNode;
   surfaceLevel: Exclude<SurfaceLevels, 1>;
   elevation: keyof Elevation;
-} & BaseBoxProps;
+} & Omit<BaseBoxProps, 'elevation'>;
```

Comment:
Shouldn't this be :
```suggestion
} & StyledPropsBlade;
```

Reply:
CardSurface is an internal component and composes CardSurfaceStyled which composes BaseBox. thus BaseBox props.